### ISSUE TITLE
Critical regression bug regarding Activate action set

### ISSUE THREAD

**Issue by AndersMalmgren** [ValveSoftware/openvr#1244]
This is a clone of https://github.com/ValveSoftware/steamvr_unity_plugin/issues/582 Since I think this is more a problem with OpenVR than unity plugin. Also its more than a week since I reported this and our game is completely broken for for Vive Wand players thanks to this regression so I hope you guys can resolve it fast.

We activate our action sets in conjunction with using prio, this code used to work fine.

```csharp
        public bool UpdateCanMove(NVRHand lastMovementHand)
        {
            var isLastMovementHand = lastMovementHand == this;
            var prio = isLastMovementHand ? 2 : 0;

            if (prio != lastPrio)
            {
                SteamVR_Actions.Movement.Activate(SteamInputSource, prio);
                lastPrio = prio;
            }

            return isLastMovementHand || SteamVR_Actions.Movement.DirectionTouched.GetActive(SteamInputSource);
        }
```

This code checks if you are already moving using this specific controller (touchpad locomotion and you can move with both left and right controller, we check which one we should move with by testing sqrt between both vector2s). 

And if you are moving it upgrades the movement action set to prio 2. This way you can grab your firearm or other item which as prio 0 on its action set and still move with the with the touchpad even if you have actions bound to touchpad that normally would conflict. 

Used to work perfect. But now each other frame `SteamVR_Actions.Movement.DirectionTouched.GetActive(SteamInputSource);` return false and the player stops, in next frame he starts to walk again. Resulting in strobe effect.  So this is a regression, I cant tell you which version it stopped working in though.

I have tried to remove `SteamVR_Actions.Movement.Activate(SteamInputSource, prio);` which fixes the problem. But now I can not pick up items without stop moving which is unacceptable solution.

We had initial problems with prio and action sets, but you guys fixed those, https://github.com/ValveSoftware/openvr/issues/1020 

You can watch the video in that post to get en explanation how we use action sets

Here is a repro in Unity with two different scenarios that break. You can find them in repro.cs

I removed SteamVR for size
[SteamVR_Repro.zip](https://github.com/ValveSoftware/steamvr_unity_plugin/files/3838409/SteamVR_Repro.zip)

But basicly calling Activate with a prio of 1 or two or higher breaks completly and GetActive returns false.

Game is completely  broken  for Vive Wand players so please if you coudl look at this ASAP we would be really happy. Thanks

**Comment by AndersMalmgren** [comment]
I have been downloading old versions of steamvr using this  method 

https://steamcommunity.com/sharedfiles/filedetails/?id=1086279994

it broke between 1.8.1 and 1.8.2 here is the manifest ids i used

17 sep 1.8.2 (broken)
250820 250821 3738881457872358261
250820 250827 7405209456960741105
250820 250824 6413409665598930737
250820 250830 8255214014436030582
250820 250833 7725354146120744174
250820 250831 8065879212735109331

16 sep  1.8.1 (working)
250820 250821 539049345897333342
250820 250827 7405209456960741105
250820 250824 4615817992650268032
250820 250830 8255214014436030582
250820 250833 7725354146120744174
250820 250831 8065879212735109331

**Comment by AndersMalmgren** [comment]
I just tried 1.9.7 and it fixes my issues with Vive. But now there are problems with Knuckles instead :/ If you try my repro from earlier with Knuckles then GetAxis will return what  ever values was present when you call Activate. So after Activate is called GetAxis will return same value no matter what actual value GetAxis should have

**Comment by AndersMalmgren** [comment]
Fixed in 1.9.12
